src/installer/installer.go

```diff
@@ -734,7 +734,38 @@ func (i *installer) cleanupInstallDevice() error {
 		return nil
 	}
 
-	vgName, err := i.ops.GetVGByPV(i.Device)
+	error := i.cleanupDevice(i.Device)
+
+	if error != nil {
+		return error
+	}
+
+	if i.ops.IsRaidMember(i.Device) {
+		i.log.Infof("A raid was detected on the device (%s) - cleaning", i.Device)
+
+		devices, error := i.ops.GetRaidDevices(i.Device)
+
+		if error != nil {
+			return error
+		}
+
+		for _, device := range devices {
+			// Cleaning the raid device itself before removing membership.
+			i.cleanupDevice(device)
```

Comment:
So now we *do* wipe all devices in the array, regardless of whether they're the installation disk or not? As discussed in your previous PR, this would have to come with a UI warning, and we have an entire epic (MGMT-7409) about disk preservation that would have to address this

Reply:
Or maybe I misunderstand. Can you please explain how the new behavior in this PR differs from the existing behavior?

Reply:
Raid device consists by multiple devices. 
In case of PV on the raid device we remove it (the PV but not wipe the raid device). 
Before this PR we stopped the raid device and remove the installation device from it which means destroying the data.
We don't wipe or remove the other disks from the raid array.
IMO destroying the RAID justifies a warning @avishayt @romfreiman @yevgeny-shnaidman  WDYT?

Reply:
Last time we agreed that if we don't touch the other disks no warning needed. @avishayt and @romfreiman had strong argument against the warning

Reply:
@omertuc anyway the warning is from the assisted-service not the installer. Let's approve this PR and wait for @avishayt response